openmls/src/messages/proposals.rs

```diff
@@ -265,7 +265,7 @@ impl From<Vec<u8>> for ExternalInitProposal {
     Debug, PartialEq, Clone, Serialize, Deserialize, TlsDeserialize, TlsSerialize, TlsSize,
 )]
 pub struct AppAckProposal {
-    received_ranges: TlsVecU32<MessageRange>,
+    pub(crate) received_ranges: TlsVecU32<MessageRange>,
```

Comment:
Not sure this change belongs in this PR. Or is there something I'm missing here?

Reply:
It's used for building this proposals in tests. I hesitated between covering it or not since anyway AppAck is going to be removed. 
I'll remove AppAck coverage, simpler

Reply:
Ah, so I did indeed miss something. I commented [here](https://github.com/openmls/openmls/issues/916#issuecomment-1318570071) to make sure that AppAck is added when/if it is implemented.